package.json

```diff
@@ -16,7 +16,6 @@
     "@okta/okta-signin-widget": "^5.7.2",
     "antd": "^4.4.3",
     "axios": "^0.21.1",
-    "bcryptjs": "^2.4.3",
     "craco": "0.0.3",
```

Comment:
These packages were removed as the PR that removed them has yet to be merged into main.